st2common/st2common/services/coordination.py

```diff
@@ -161,22 +192,44 @@ def coordinator_setup():
         # to work
         coordinator = NoOpDriver(member_id)
 
-    coordinator.start()
+    coordinator.start(start_heart=start_heart)
     return coordinator
 
 
 def coordinator_teardown(coordinator):
     coordinator.stop()
 
 
-def get_coordinator():
+def get_coordinator(start_heart=False, use_cache=True):
+    """
+    :param start_heart: True to start heartbeating process.
+    :type start_heart: ``bool``
+
+    :param use_cache: True to use cached coordinator instance. False should only be used in tests.
+    :type use_cache: ``bool``
+    """
     global COORDINATOR
 
     if not configured():
         LOG.warn('Coordination backend is not configured. Code paths which use coordination '
                  'service will use best effort approach and race conditions are possible.')
 
+    if not use_cache:
+        coordinator = coordinator_setup(start_heart=start_heart)
+        return coordinator
```

Comment:
return in one line?  `return coordinator_setup(start_heart=start_heart)`

Reply:
That's a personal preference - I can prefer return in a separate line.

Reply:
Ok. My suggestion here is because we are not doing anything with the variable `coordinator` so why not just return here.  This is also 1 line of code less.  What's the reasoning for your preference?